tasks/section1.yml

```diff
@@ -1019,6 +1019,7 @@
       regexp: '(selinux|enforcing)\s*=\s*0\s*'
       follow: yes
   register: selinux_grub_patch
+  ignore_errors: yes
```

Comment:
I'd prefer if we didn't use `ignore_errors: yes` on PATCH tasks in order to pass the CI test. This was functioning when I tested it manually, is this because the Travis tests are running against a Docker container?

Also if something fails then I'd like to figure out why and either refine the task or write up some conditional logic to skip it without using ignore_errors.